README.md

```diff
@@ -13,12 +13,15 @@ Prerequisites: JDK 11, update Intellij to the most recent version.
    1. If there are any further prompts, accept the defaults.
```

Comment:
Would suggest adding more information in your README on how your software works and more clear steps on how one should download it